webpack.dev.js

```diff
@@ -34,25 +27,16 @@ module.exports = merge.smart(require('./webpack.base.js'), {
     ],
   },
   plugins: [
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('development'),
-      },
-    }),
     new webpack.DefinePlugin({
       NETLIFY_CMS_VERSION: JSON.stringify(require("./package.json").version + "-dev"),
     }),
     new webpack.NoEmitOnErrorsPlugin(),
-    new ExtractTextPlugin({
-      filename: '[name].css',
-      disable: true,
-    }),
   ],
   devtool: 'source-map',
-  devServer: {
-    contentBase: 'example/',
-    historyApiFallback: true,
-    disableHostCheck: true,
```

Comment:
I dropped `historyApiFallback` and `disableHostCheck`, is there a reason they were needed?

Reply:
I added them back on the PR to this branch just submitted, we should test to see if we can get rid of them